src/models/files/ZosPackageFile.js

```diff
@@ -0,0 +1,113 @@
+import _ from 'lodash'
+import ZosNetworkFile from './ZosNetworkFile'
+import { Logger, FileSystem as fs } from 'zos-lib'
+
+const log = new Logger('ZosPackageFile')
+
+export default class ZosPackageFile {
+
+  constructor(fileName = 'zos.json') {
+    this.fileName = fileName
+    this.data = fs.parseJsonIfExists(this.fileName) || {};
+  }
+
+  exists() {
+    return fs.exists(this.fileName)
+  }
+
+  ifMissingThrow(msg) {
```

Comment:
This is more a subjective comment, but I don't like this kind of methods. I'd rather have a caller do `if (!packageFile.exists()) throw Error(msg)` rather than add an extra method here that clutters the interface.